public/src/components/common/button_objs.jsx

```diff
@@ -1,3 +1,19 @@
+function statusToggleActivePause(status) {
+  if (status === 'draft' || status === 'pause') {
+    return ['Activate', 'active', 'success', false];
```

Comment:
Maybe these should be maps/objects? would make this logic a bit cleaner to read... also potentially less buggy (someone could modify the order here and the component would break.